onnxruntime/core/graph/schema_registry.cc

```diff
@@ -194,6 +194,23 @@ DomainToVersionMap SchemaRegistryManager::GetLatestOpsetVersions(bool is_onnx_on
   return domain_version_map;
 }
 
+static bool IsDomainVersionBeyondSupportedRange(
+    const std::string& domain,
+    const int op_set_version) {
+  // check the ONNX schema registry
+  auto& onnx_domain_version_map =
+      ONNX_NAMESPACE::OpSchemaRegistry::DomainToVersionRange::Instance().Map();
+
+  auto it = onnx_domain_version_map.find(domain);
+  if (it != onnx_domain_version_map.end() && op_set_version > it->second.second) {
+    // The domain is beyond what is registered.
+    return true;
```

Comment:
```return (it != onnx_domain_version_map.end() && op_set_version > it->second.second);``` should suffice, right?

Reply:
@fdwr Can you take a look here too? Thanks!

Reply:
@pranavsharma : Yep, equivalent and sufficient.